company.rb

```diff
@@ -0,0 +1,65 @@
+$LOAD_PATH << '.'
+require 'quality-assurance'
+require 'human-resource'
+require 'employee'
+require 'module'
+
+
+class Company
+include Read_CSV_File
+  attr_accessor :company_id, :company_name, :company_type, :headquarter, :founded
+  @@no_of_employees = 0
```

Comment:
this should be an instance method. Because number of employees is applicable per company object. 
If you had multiple companies then this variable would store the total employees of all the companies.